data/campaigns/World_Conquest/gui/settings_dialog.cfg

```diff
@@ -109,7 +109,7 @@
                         [column]
                             grow_factor = 1
                             [label]
-                                label = "length"
+                                label = _ "length"
```

Comment:
Examining the map-generator code, the meaning of this appears to be "approximate width and height of the map (which will be roughly square)". It's not ready for translation, because the English doesn't match the meaning.

Reply:
Maybe it should be "size"?